templates/CSD-1.html.tmpl

```diff
@@ -236,200 +338,405 @@
                 {{/each}}
-             </table>
-			 {{else}}
+			{{else}}
              <p><em>N/A</em></p>
-             {{/if}}
+            {{/if}} 
+
 
         <h3 id="section_4_2"><span>4.2</span>Procedural Constraints</h3>
-        {{#if data.requirements.proceduralconstraints}}
-			<table>
-                <tr>
-                    <th>No.</th>
-                    <th>Document</th>
-                    <th>Requirement</th>
-                    <th>Applicability</th>
-                </tr>
-                {{#each data.requirements.proceduralconstraints}}
-                <tr>
-                    <td class="lean" rowspan="3">{{key}}</td>
-                    <td class="lean">URS</td>
-                    <td class="content-wrappable">Summary: {{{ursName}}}{{#if ursDescription}}<br>{{{ursDescription}}}{{/if}}</td>
-                    <td class="lean">{{applicability}}</td>
-                </tr>
-                <tr>
-                    <td class="lean">FS</td>
-                    <td class="content-wrappable">Summary: {{{fsName}}}{{#if }}<br>{{{fsDescription}}}{{/if}}</td>
-                    <td class="lean">{{applicability}}</td>
-                </tr>
-                <tr>
-                    <td class="lean">CS</td>
-                    <td class="content-wrappable">N/A</td>
-                    <td class="lean">N/A</td>
-                </tr>
+            {{#if data.requirements.proceduralconstraints}}
+                {{#if data.requirements.proceduralconstraints.noepics}}
+                <table>
+                    <tr>
+                        <th>No.</th>
+                        <th>Document</th>
+                        <th>Requirement</th>
+                        <th>Applicability</th>
+                    </tr>
+                    {{#each data.requirements.proceduralconstraints.noepics}}
+                    <tr>
+                        <td class="lean" rowspan="3">{{key}}</td>
+                        <td class="lean">URS</td>
+                        <td class="content-wrappable">Summary: {{{ursName}}}{{#if ursDescription}}<br>{{{ursDescription}}}{{/if}}</td>
+                        <td class="lean">{{applicability}}</td>
+                    </tr>
+                    <tr>
+                        <td class="lean">FS</td>
+                        <td class="content-wrappable">Summary: {{{fsName}}}{{#if }}<br>{{{fsDescription}}}{{/if}}</td>
+                        <td class="lean">{{applicability}}</td>
+                    </tr>
+                    <tr>
+                        <td class="lean">CS</td>
+                        <td class="content-wrappable">N/A</td>
+                        <td class="lean">N/A</td>
+                    </tr>
+                    {{/each}}
+                </table>
+                {{/if}}      
+
+                {{#each data.requirements.proceduralconstraints.epics}}
+                <h4 id="section_4_2_{{epicIndex}}"><span>4.2.{{epicIndex}}</span>{{epicName}}</h4>
```

Comment:
What does epicIndex mean?? Is it the key? I suppose not. 

I would recommend to add the epic key from Jira somewhere, since it is what the document history will show

Reply:
epicIndex is to provide an index of the chapters/subchapters f.e.:
- Having the stories that belongs to n epics we display the stories grouped by epic having there:
  4.2: Table with Stories not linked to epics
  4.2.1 Epic Name/summary
        Table of Stories linked with the epic
  4.2.2 Epic Name/summary
        Table of Stories linked with the epic

 As you said Martin, perhaps it would be necessary to provide in the title the Epic Id